controllers/extensions/addon_controller_stages.go

```diff
@@ -1227,6 +1236,15 @@ func validateVersion(annotations, kbVersion string) (bool, error) {
 	return validate, nil
 }
 
+func checkAddonSpec(addon *extensionsv1alpha1.Addon) error {
+	if addon.Spec.Type == constant.AddonHelmSpecType {
```

Comment:
Better to use this constant https://github.com/apecloud/kubeblocks/blob/1fe340139225f421c5330064aacae50219440eb7/apis/extensions/v1alpha1/type.go#L25-L25